reference/Uno.Extensions.Commerce/Commerce.macOS/Commerce.macOS.csproj

```diff
@@ -73,15 +75,24 @@
     <DefineConstants>$(DefineConstants);WINUI</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Uno.WinUI" Version="4.5.0-dev.562" />
-    <PackageReference Include="Uno.WinUI.RemoteControl" Version="4.5.0-dev.562" Condition="'$(Configuration)'=='Debug'" />
-    <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.5.0-dev.562" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces">
+      <Version>6.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.Memory">
+      <Version>4.5.5</Version>
+    </PackageReference>
+    <PackageReference Include="Uno.Toolkit.UI">
```

Comment:
This reference needs to be using the WinUI package

Reply:
Actually you can just remove this reference, there's a reference to the WinUI Toolkit package just below this one

Reply:
@kazo0 , I removed the Uno.Toolkit.UI got this error on CI:

![image](https://user-images.githubusercontent.com/5515535/197227197-36a04752-ffba-49ac-ba77-6fb1f5f1fe52.png)

Then I put it again and back to the first error "Cannot build with both Uno.WinUI and Uno.UI nuget packages referenced"

Reply:
@kazo0 , thanks for all the help!